terraform/providers.tf

```diff
@@ -3,29 +3,38 @@
 # used to assume the roles required to access remote state in the
```

Comment:
Are we intentionally leaving `default_tags` blocks off of the `images_production_thirdparty` and `images_staging_thirdparty` providers?

Reply:
No, that was an oversight on my part.  Please see commit 3f6b018.